grammar.js

```diff
@@ -293,6 +299,8 @@ module.exports = grammar({
                         alias($.inline_math_close, "_lowercase"),
                         alias($.variable_open, "_lowercase"),
                         alias($.variable_close, "_lowercase"),
+                        alias($.inline_link_target_open, "_lowercase"),
+                        alias($.inline_link_target_close, "_lowercase"),
                         alias($.link_description_begin, "_lowercase"),
                         alias($.link_description_end, "_lowercase"),
```

Comment:
As you can see here, in the `_verbatim_segment` all open and closing markup as well as link beginnings and ends are disabled. This is correct as it is